docs/source/test_set.rst

```diff
@@ -34,4 +35,22 @@ To run the test set on a pre-trained model, use this method.
     trainer.test(model)
 
 In this  case, the options you pass to trainer will be used when
-running the test set (ie: 16-bit, dp, ddp, etc...)
\ No newline at end of file
+running the test set (ie: 16-bit, dp, ddp, etc...)
+
+
+Test additional data loaders
+----------------------------
+You can still run inference on a test set even if the `test_dataloader` method hasn't been
```

Comment:
```suggestion
You can still run inference on a test set even if the `test_dataloaders` method hasn't been
```

Reply:
I changed it back because the LightningModule methods is singular (even if you can return multiple)

Reply:
hope it's ok